projects/plugins/wpcomsh/composer.json

```
@@ -0,0 +1,144 @@
+{
+	"name": "automattic/jetpack-wpcomsh",
+	"description": "A helper for connecting WordPress.com sites to external host infrastructure.",
+	"type": "wordpress-plugin",
+	"license": "GPL-2.0-or-later",
+	"require": {
+		"php": ">=7.4",
+		"automattic/at-pressable-podcasting": "^2.0",
+		"automattic/custom-fonts": "^3.0",
+		"automattic/custom-fonts-typekit": "^2.0",
+		"automattic/text-media-widget-styles": "^2.0",
+		"automattic/wc-calypso-bridge": "2.4.1",
```

Somehow this got changed compared to what was in wpcomsh.
```suggestion
		"automattic/wc-calypso-bridge": "2.5.1",
```

---

Must have been a bad merge, thanks!